.env.development

```diff
@@ -12,7 +12,7 @@ NEXT_PUBLIC_EIP721_SUBGRAPH="https://api.thegraph.com/subgraphs/name/sunguru98/e
 NEXT_PUBLIC_NFT_SALES_SUBGRAPH="https://api.thegraph.com/subgraphs/name/adamgobes/nft-sales-indexer"
 NEXT_PUBLIC_NFT_PORT_API_KEY="nft-port-api-key-here"
 NEXT_PUBLIC_NOTIFICATION_REQ_MESSAGE="signing notification request"
-DATABASE_URL="postgresql://user:password@localhost:5432/notifications?schema=public"
+DATABASE_URL="postgres://backed_notifications_rinkeby_user:2vl9ZQ1RCU1qfWMXLBFshtOOR4z90za5@dpg-c9a6lr319n0a05tsiq8g-a.oregon-postgres.render.com/backed_notifications_rinkeby"
```

Comment:
in dev should connect to development DB, this is not really related to this PR but long overdue